docs/replication/group-replication-flow-control.md

```diff
@@ -0,0 +1,18 @@
+# Manage Group Replication flow control
+
+In replication, flow control prevents one member from falling too far behind the cluster and avoid excessive buffering. A cluster is not required to keep members in sync together for replication. The pending transactions in the relay log only increase for the lagging replica. 
+
+Flow control helps these members by setting thresholds for transactions to certify and transactions to apply. If the thresholds are exceeded, and during the duration that they are exceeded, flow control adjusts the writer members to the capacity of the delayed member. This action ensures that all members are in sync. Each member sends statistics to the group.
+
+Flow controls work asynchronously and depends on the following:
+
+* Monitoring the throughput and queue sizes of each member
+* Throttling members to avoid writing beyond the capacity available
+
+The following system variables set flow control behavior in Group Replication:
+
+* [group_replication_flow_control_mode](https://dev.mysql.com/doc/refman/8.0/en/group-replication-options.html#sysvar_group_replication_flow_control_mode)
+* [group_replication_flow_control_certifier_threshold](https://dev.mysql.com/doc/refman/8.0/en/group-replication-options.html#sysvar_group_replication_flow_control_certifier_threshold)
+* [group_replication_flow_control_applier_threshold](https://dev.mysql.com/doc/refman/8.0/en/group-replication-options.html#sysvar_group_replication_flow_control_applier_threshold)
+
+Flow control is enabled and disabled by selecting a value in the group_replication_flow_control_mode variable. Flow control can also be enabled on the certifier or applier level or both and sets the threshold level in seconds. 
```

Comment:
Not in seconds, but in number of transactions waiting in the queue.

Reply:
removed.